pkg/convert/pot.go

```diff
@@ -0,0 +1,144 @@
+package convert
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/pot"
+)
+
+type poTag string
+
+const (
+	MsgId  poTag = "msgid"
+	MsgStr poTag = "msgstr"
+)
+
+func ToPot(m model.Messages) ([]byte, error) {
+	var b bytes.Buffer
+
+	if _, err := fmt.Fprintf(&b, "\"Language: %s\n", m.Language); err != nil {
+		return nil, fmt.Errorf("write language: %w", err)
+	}
+
+	for i, message := range m.Messages {
+		if i > 0 {
+			if _, err := fmt.Fprint(&b, "\n"); err != nil {
+				return nil, fmt.Errorf("write new line: %w", err)
+			}
+		}
+
+		if message.Description != "" {
+			if _, err := fmt.Fprintf(&b, "#. %s\n", message.Description); err != nil {
```

Comment:
If we have a multiline description, doesn't it break **po**? Do we have to escape special characters in the description?